build/azure-pipelines/darwin/product-build-darwin.yml

```diff
@@ -1,7 +1,7 @@
 steps:
 - task: NodeTool@0
   inputs:
-    versionSpec: "8.12.0"
+    versionSpec: "10.15.1"
```

Comment:
looks like GCI is failing because we need to bring this change into our azure-pipelines scripts too